### ISSUE TITLE
About the implementation.

### ISSUE THREAD

**Issue by jingmouZhong** [MMIRMagPie/DualVGR-VideoQA#2]
Hello, thanks for you great works! In Part B of Section 3 of your paper, you mentioned that you use appearance-motion correlation graph and motion-appearance correlation graph to get embedding ![](http://latex.codecogs.com/svg.latex?Z^{(t)}_{CA}) and ![](http://latex.codecogs.com/svg.latex?Z^{(t)}_{CM}). In your code model/models.py lines 95-100 and 150-160, the implementation of ![](http://latex.codecogs.com/svg.latex?Z^{(t)}_{CA}) and ![](http://latex.codecogs.com/svg.latex?Z^{(t)}_A) is the same, such as ![](http://latex.codecogs.com/svg.latex?Z^{(t)}_{CM}) and ![](http://latex.codecogs.com/svg.latex?Z^{(t)}_M). If they are the same, why are the two diagrams described in the paper? Meanwhile, due to limited GPUs, I run the experiment at MSVD-QA Datasets but only got 38.25 accuracy. 
Looking forward to your reply, thank you.

**Comment by MM-IR** [comment]
Hi, in each visual channel, the implementations of our two-branch graphs are the same, but they have different weights. I am not sure that If there is something wrong here, but the acc is about 39... You should adjust the number of epochs or something like that.

**Comment by MM-IR** [comment]
Furthermore, I guess that do you just use the training part? If so, that is the dev set.

**Comment by MM-IR** [comment]
Instead, you have to use our test set to obtain the final results.

**Comment by jingmouZhong** [comment]
![1641437502(1)](https://user-images.githubusercontent.com/48832539/148320620-6cd47e93-a1c3-4527-ac97-8026a4cb8efc.png)
[2021-12-15_Wednesday_14_39_34DualVGR_stdout.log](https://github.com/NJUPT-MCC/DualVGR-VideoQA/files/7819160/2021-12-15_Wednesday_14_39_34DualVGR_stdout.log)

**Comment by jingmouZhong** [comment]
I run train.py and get log file and run validate.py but I only can get 38.25%.

**Comment by MM-IR** [comment]
Do you have the right Configuration? Sorry for my limited time, and if I have enough time, I can check it for you step by step.

**Comment by jingmouZhong** [comment]
![1641459320(1)](https://user-images.githubusercontent.com/48832539/148356424-0ba03caa-ca9b-48dd-88dc-c0243bacb090.png)
![1641459119(1)](https://user-images.githubusercontent.com/48832539/148356050-2ba17c02-03dd-453b-b52f-8b25d3e0cf49.png)

**Comment by MM-IR** [comment]
Thanks for your reply. Do you try other datasets? (not sure about your problem, nothing wrong?) I am just busy with my study, thus I cannot run the experiments for you. However, in my previous experiments, this configuration actually leads to 39% acc. Best wishes.

**Comment by MM-IR** [comment]
I guess that I have understood the difference here: Random seeds just guarantee reproducible results for a specific platform, device, and PyTorch release, but not across different machines.
